srcpkgs/dialog/template

```diff
@@ -3,18 +3,26 @@ pkgname=dialog
 _distver=1.3
 _date=20200327
 version="${_distver}.${_date}"
-revision=1
+revision=2
```

Comment:
No need to bump, as it isn't in the repo; Only time it is needed to change is when it is in repo and needs a rebuild.